staging/src/k8s.io/client-go/plugin/pkg/client/auth/gcp/gcp.go

```diff
@@ -120,7 +120,7 @@ func newGCPAuthProvider(_ string, gcpConfig map[string]string, persister restcli
 	// this should be updated to use klog.Warningf in v1.24 to more actively warn consumers
 	warnOnce.Do(func() {
 		klog.V(1).Infof(`WARNING: the gcp auth plugin is deprecated in v1.22+, unavailable in v1.25+; use gcloud instead.
```

Comment:
switch to klog.Warningf as well here and remove the todo above?

Reply:
Done.